panels/models/attraction.py

```diff
@@ -33,6 +34,30 @@ def sluggify(s):
     return RE_SLUG.sub('-', s).lower().strip('-')
 
 
+def treeify(items, keys, value_key=None):
+    if not keys:
+        return items
+    keys = listify(keys)
+    last_key = keys[-1]
+    treeified = OrderedDict()
+    for item in items:
+        current = treeified
+        for key in keys:
+            attr = key(item) if callable(key) else getattr(item, key)
+            if attr not in current:
+                current[attr] = [] if key is last_key else OrderedDict()
+            current = current[attr]
+        if value_key:
+            if callable(value_key):
```

Comment:
It's a little strange that you did this all on one line for `attr` and then did it differently here. For the record, I prefer the one line version.

Reply:
Yeah, that was because of my self-imposed 79 character limit. I'll rework it to be on one line!